### PR TITLE
Task4394.dev Was changed twig mini-pager(added arrows) reworks block "Fashion"- changes display mode

### PR DESCRIPTION
**Pull request by Dexs1**
https://redmine.devbranch.work/issues/4394

### CODE REVIEW

**Comment by Dexs1** [comment]
And also added some styles to blocks that were themed earlier for correct rendering.

File: web/themes/custom/kitrab/src/scss/_grouprecent.scss [hunk 1]
```diff
@@ -0,0 +1,14 @@
+.medium-block {
+  //.image-style-medium-content {
+  //    width: 100%;
+  //}
```
Comment (devbranch-vitaliy) [inline web/themes/custom/kitrab/src/scss/_grouprecent.scss]: Dead code
```suggestion
```

File: web/themes/custom/kitrab/src/scss/_life-news.scss [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  .image-style-medium-content {
+    min-height: 150px;
+  }
+  .field--name-title {
+    font-size: rem(12);
+    h2 {
+      line-height: rem(18);
+    }
+  }
+}
```
Comment (devbranch-vitaliy) [inline web/themes/custom/kitrab/src/scss/_life-news.scss]: Need a blank space at the end of the file.
```suggestion
}

```

File: web/themes/custom/kitrab/src/scss/_mainpage.scss [hunk 3]
```diff
@@ -0,0 +1,6 @@
+.layout__region.layout__region--first {
+    .block {
+      margin-right:rem(20);
+      margin-left:rem(20);
+    }
+}
```
Comment (devbranch-vitaliy) [inline web/themes/custom/kitrab/src/scss/_mainpage.scss]: Need a blank space at the end of the file.
```suggestion
}

```

File: web/themes/custom/kitrab/src/scss/app.scss [hunk 4]
```diff
@@ -1,4 +1,8 @@
+@import "global";
+@import "mainpage";
 @import "grouprecent";
 @import "breackpoints";
 @import "musicfastival";
-@import "global";
+@import "life-news";
+
+
```
Comment (devbranch-vitaliy) [inline web/themes/custom/kitrab/src/scss/app.scss]: At the end of the file should be only 1 blank space.
```suggestion
```

**Review by devbranch-vitaliy — CHANGES_REQUESTED** [review]

**Review by devbranch-vitaliy — APPROVED** [review]

**Pull request merged**